authors/apps/bookmarks/serializers.py

```diff
@@ -0,0 +1,65 @@
+from rest_framework import serializers
+from authors.apps.profiles.models import Profile
+from authors.apps.articles.serializers import ArticleSerializer
+from authors.apps.profiles.serializers import GetProfileSerializer
+from .models import Bookmark
+
+
+class BookmarkSerializer(serializers.ModelSerializer):
+    """
+    BookmarkSerializer class to validate, transform bookmark datatypes
+    """
+    profile = serializers.SerializerMethodField()
+    article = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Bookmark
+        fields = ('id', 'profile', 'article', 'created_at')
+
+    def get_profile(self, obj):
```

Comment:
- Seeing as one can only get there own bookmarks, is there a need to have the requester's profile included in the response?

Reply:
Updated. Now we only return article and bookmark details instead of profile details as well.